README.md

```diff
@@ -62,7 +69,7 @@ Underneath it uses Scala macros to give you:
 To include Chimney to your SBT project, add the following line to your `build.sbt`:
 
 ```scala
-libraryDependencies += "io.scalaland" %% "chimney" % "0.6.2"
+libraryDependencies += "io.scalaland" %% "chimney" % chimneyVersion
 ```
 
 Library is released for Scala 2.12.x and 2.13.x.
```

Comment:
TODO: change to "is supported for Scala 2.12.x and 2.13.x" since we have a 2.11.x version as well.